store/store.go

```diff
@@ -15,9 +16,9 @@ const (
 // Params is the parameters to use for the Store object
 type Params struct {
 	// Kv is the bootstrap kvdb instance
-	Kv            kvdb.Kvdb
+	Kv kvdb.Kvdb
```

Comment:
nit: The older formatting looked correct. Could you please keep it aligned like the previous version?